akka-actor/src/main/scala/akka/dispatch/AbstractDispatcher.scala

```diff
@@ -26,7 +26,7 @@ final case class Envelope private (val message: Any, val sender: ActorRef)
 
 object Envelope {
   def apply(message: Any, sender: ActorRef, system: ActorSystem): Envelope = {
-    if (message == null) throw InvalidMessageException("Message is null")
+    if (message == null) throw InvalidMessageException(s"Message sent from $sender is null.")
```

Comment:
Perhaps:
```suggestion
    if (message == null) throw InvalidMessageException(s"Message sent from [$sender] is null.")
```

Reply:
with Akka Typed there is no sender (and in some cases in untyped as well) so it would be nice to use different messages depending on if `sender` is null or not

Reply:
OK

Reply:
> Perhaps:

OK